rust/sdk-processor/src/config/processor_config.rs

```diff
@@ -155,9 +163,9 @@ pub struct ParquetDefaultProcessorConfig {
     pub max_buffer_size: usize,
     #[serde(default = "ParquetDefaultProcessorConfig::default_parquet_upload_interval")]
     pub parquet_upload_interval: u64,
-    // list of table names to backfill. Using HashSet for fast lookups, and for future extensibility.
+    // Set of table name to backfill. Using HashSet for fast lookups, and for future extensibility.
     #[serde(default)]
-    pub tables: HashSet<String>,
+    pub backfill_table: HashSet<String>,
```

Comment:
I am thinking backfill_config is the right place to place this field, but I don't want to confuse SDK users with this.

Reply:
that makes sense to me too. The configs are on the implementation though right? Not in the SDK?

Reply:
yeah, but we still have clients running our processors. I am fine with adding new field to backfillConfig `parquet_backfill_table: Option<String>` @rtso wdyt?

Reply:
we can leave the config in the parquet processor config or add a good comment after adding it into the backfill config.